src/main/java/org/jabref/logic/openoffice/OOBibStyle.java

```diff
@@ -122,6 +122,7 @@
     private File styleFile;
     private long styleFileModificationTime = Long.MIN_VALUE;
     private String localCopy;
+    private boolean setDefault;
```

Comment:
Could you rename the variable to `setDefaultLayout` or smething?

Currently, the name is not that readable.

Reply:
Yes yes yes, I knew it. In the pipeline